.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @kraken-tech/open-source-maintainers
```

Comment:
Not sure why this is invalid, it's fine in [this repo](https://github.com/kraken-tech/timezone-tools/blob/main/.github/CODEOWNERS).

Reply:
It shows valid for me?

Reply:
Oh great, it is now. I realised there were some github set up steps I needed to complete, as documented in the cookiecutter readme. https://github.com/octoenergy/cookiecutter-kraken-package?tab=readme-ov-file#configure-github